curvefs/conf/client.conf

```diff
@@ -153,6 +153,8 @@ s3.threadScheduleInterval=3
 s3.cacheFlushIntervalSec=5
 s3.writeCacheMaxByte=838860800
```

Comment:
add a comment for this config, tell users they should not config a value less than 8M.

Reply:
Fixed.